packages/core/src/Stores/notification-store.js

```diff
@@ -621,6 +650,14 @@ export default class NotificationStore extends BaseStore {
             : [...this.notifications.filter(notifs => notifs.is_persistent)];
     }
 
+    removeTradeNotifications(id) {
+        if (id) {
+            this.trade_notifications = this.trade_notifications.filter(item => item.id !== id);
+        } else {
+            this.trade_notifications = [];
```

Comment:
Is it  bad practice to use `ternary operator` for adding value to the `this.trade_notifications`?

Reply:
@kate-deriv sure, I can replace it with ternary :)